challenge-07/challenge-07.js

```diff
@@ -1,51 +1,94 @@
 /*
 Crie um array com 5 items (tipos variados).
 */
-// ?
+
+var arr = [ 1, 'Max', true, null, { carro: 'verde' }]
 
 /*
 Crie uma função chamada `addItem`, que irá adicionar itens no array criado.
 A função deverá retornar o array atualizado.
 */
-// ?
+
+function addItem (item) {
+  arr.push (item);
```

Comment:
Quando for utilizar funções, procure deixar o parênteses colado no nome da função. Nesse caso, `arr.push(item)` :)